### PR TITLE
Git Hooks persistent test scenario

### PR DESCRIPTION
**Pull request by jakubschwan**
Test needs add output check.
Waiting for 7.1 release because of [RHPAM-624](https://issues.jboss.org/browse/RHPAM-624)

### CODE REVIEW

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java [hunk 1]
```diff
@@ -150,6 +166,46 @@ public void testWorkbenchProjectPersistence() {
         assertThat(containersResponse.getResult().getContainers().get(0).getContainerId()).isEqualTo(CONTAINER_ID);
     }
 
+    @Test
+    public void testGitHooksPersistence() {
+        // get git hook dir location
+        String gitHooksRemoteDir = "/opt/eap/standalone/data/bpmsuite/git/hooks"; //TODO for 7.1 bpmsuite -> kie
+
+        updateWorkbenchDeployment(gitHooksRemoteDir);
```
Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java]: Will git hooks environment variable be part of template parameters?

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java [hunk 2]
```diff
@@ -150,6 +166,46 @@ public void testWorkbenchProjectPersistence() {
...
+
+        logger.debug("Filter instances for second deployment");
+        List<String> workbenchInstanceNames = workbenchDeployment.getInstances().stream().map(Instance::getName).filter(name -> name.contains("-2-")).collect(Collectors.toList());
```
Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java]: What is the purpose of this filtering? Does it mean that getInstances() returns also pods which are not active?

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java (hunk 1 above)
Comment (jakubschwan) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java]: Not decided yet

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java (hunk 2 above)
Comment (jakubschwan) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchPersistenceIntegrationTest.java]: To make sure, that deployment was redeployed and we are working with updated pod. There is small chance, we can get active pod, where is undeploying previous configuration. But I think that filtering can be removed for now.

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
resolved conflicts

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchGitHooksPersistenceIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    public void setUp() {
+        workbenchDeployment = deploymentScenario.getWorkbenchDeployments().get(0);
+        workbenchClient = WorkbenchClientProvider.getWorkbenchClient(workbenchDeployment);
+    }
+
+    @Test
+    public void testGitHooksPersistence() {
+        List<String> workbenchInstanceNames = workbenchDeployment.getInstances().stream().map(Instance::getName).collect(Collectors.toList());
+        if (workbenchInstanceNames.isEmpty()) {
+            throw new RuntimeException("Workbench was not correctly deploy.");
```
Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/WorkbenchGitHooksPersistenceIntegrationTest.java]: Is this needed? AFAIK the scenario itself should check if Workbench is scaled when it is deployed.

**Pull request merged**